CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Networking/DocumentServiceProtocol.swift

```diff
@@ -18,35 +18,13 @@ public protocol DocumentServiceProtocol: AnyObject {
     var analysisCancellationToken: CancellationToken? { get set }
     
     func cancelAnalysis()
-    func remove(document: GiniCaptureDocument)
-    func resetToInitialState()
+    func remove(document: GiniCaptureDocument) // delete
```

Comment:
Whoops. I committed these trailing comments by accident.

Reply:
I'll remove the comments